olp-cpp-sdk-dataservice-write/include/olp/dataservice/write/VolatileLayerClient.h

```diff
@@ -89,9 +90,20 @@ class DATASERVICE_WRITE_API VolatileLayerClient {
 
   /**
    * @brief Cancel all pending requests.
+   * @deprecated Use \ref CancelPendingRequests intead.
    */
+  OLP_SDK_DEPRECATED("Use CancelPendingRequests instead.")
   void CancelAll();
 
+  /**
+   * @brief Cancels all the ongoing operations that this client started.
+   *
+   * Returns instantly and does not wait for the callbacks.
+   * Use this operation to cancel all the started requests without
+   * destroying the actual client instance.
+   */
```

Comment:
Same as above. Consider mentioning that the method canceling queued and ongoing requests.